django_mesh/tests/test_models.py

```diff
@@ -35,7 +46,72 @@ def test_has_custom_summary(self):
         self.assertNotEqual(self.p1.summary, self.p1.teaser)
         self.assertEqual(self.p1.summary, self.p1.custom_summary)
 
-    def test_render_stays_the_same_when_no_links(self):
+    def test_markup(self):
+        self.c1.save()
+
+        self.p1.channel = self.c1
+        self.p7.channel = self.c1
+        self.p8.channel = self.c1
+
+        self.p1.text = 'hello'
+        self.p7.text = 'hello'
+        self.p8.text = 'hello'
+
+        self.p1.save() # simple
+        self.p7.save() #markdown
+        self.p8.save() #textile
+
+        expected_html_markup_and_textile = """<p>hello</p>"""
+        expected_html_simple = """hello"""
+
+
+        self.assertHTMLEqual(expected_html_simple, self.p1.rendered_text)
+        self.assertHTMLEqual(expected_html_markup_and_textile, self.p7.rendered_text)
+        self.assertHTMLEqual(expected_html_markup_and_textile, self.p8.rendered_text)
+
+    @patch('requests.sessions.Session.request')
+    def test_check_if_markdown_automatically_wrap_plain_URLs_in_anchor_tags(self, mock_request):
+        youtube = get_mock_no_oembed()
+        mock_request.return_value = Mock()
```

Comment:
Do the get_mock* functions return Mock objects? If so, there's no need to copy their values to a new Mock.